src/librustdoc/config.rs

```diff
@@ -24,6 +25,33 @@ use crate::opts;
 use crate::passes::{self, Condition, DefaultPassOption};
 use crate::theme;
 
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+pub enum OutputFormat {
+    Json,
+    HTML,
+}
+
+impl OutputFormat {
+    pub fn is_json(&self) -> bool {
+        match *self {
```

Comment:
`OutputFormat` is `Copy` so you can just take `self` here and save on the `*self` in the match.